package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bump-cli",
   "description": "The Bump CLI is used to interact with your API documentation hosted on Bump by using the API of developers.bump.sh",
-  "version": "2.0.0-beta",
+  "version": "2.0.0-beta.1",
```

Comment:
When do you plan to remove the `beta` here?

Reply:
Sorry this commit was not pushed from friday. It should have been in `main` branch already. I've updated the PR.

The beta will be removed when we release 2.0.0